auth-web/src/components/auth/common/SubProductSelector.vue

```diff
@@ -0,0 +1,108 @@
+<template>
+  <v-form
+    id="sub-product-selector"
+    class="pl-1 pr-3"
+    ref="productSelectorFormRef"
+  >
+    <p>Select the access type you want (Required):</p>
+    <v-radio-group
+      hide-details
+      v-model="selectedProduct"
+      class="sub-product-radio-group"
+    >
+      <div
+        v-for="subProduct in subProductConfig"
+        :key="subProduct.type"
+        class="sub-product-radio-wrapper ml-n9"
+        :class="{'selected' : selectedProduct === subProduct.type }"
+      >
+        <v-divider />
+        <v-radio
+          class="sub-product-radio-btn mt-6 ml-9"
+          :value="subProduct.type"
+        >
+          <template v-slot:label>
+            <v-row no-gutters>
+              <v-col cols="12">
+                <v-label>{{ subProduct.label }}</v-label>
+              </v-col>
+              <v-col class="mt-1">
+                <p>
+                  <ul>
+                    <li v-for="(bullet, index) in subProduct.productBullets" :key="index" class="bullet mt-2 ml-n1">
+                      <span :class="{ 'font-weight-bold': isImportantBullet(subProduct, index) }">
+                        {{ bullet }}
+                      </span>
+                    </li>
+                  </ul>
+                </p>
+                <p v-if="subProduct.note" class="sub-product-note my-6 pr-3">
+                  <strong>Note:</strong> <span v-html="subProduct.note"></span>
```

Comment:
v-html should be fine I believe we control subProduct.note entirely

Reply:
You bet, it's entirely local configuration 👍

Reply:
@seeker25 Do you think we should sanitize here? 
Does Auth Web have a package or tool to do this already perhaps?

Reply:
No need, we have control over subProduct.note - I don't think I added it to auth-web